services/provider/server/server.go

```diff
@@ -1468,6 +1468,17 @@ func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStat
 		return nil, status.Errorf(codes.Internal, "Failed to construct status response: %v", err)
 	}
 
+	clientOperatorVersion, err := semver.Parse(req.ClientOperatorVersion)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Malformed ClientOperatorVersion: %v", err)
+	}
+	consumerProcessed, exist := storageConsumer.GetAnnotations()[util.StorageConsumer418ProcessedAnnotationKey]
+	if clientOperatorVersion.Major == 4 &&
+		clientOperatorVersion.Minor == 18 &&
+		(!exist || consumerProcessed == util.StorageConsumer418ProcessedAnnotationPending) {
+		channelName = strings.Replace(channelName, "4.19", "4.18", 1)
+	}
+
```

Comment:
Please remove any annotation logic from this PR. We will introduce similar logic based on feedback coming from the upgrade controller implementation

Reply:
removed.